core/drivers/crypto/caam/utils/utils_mem.c

```diff
@@ -51,21 +51,27 @@
 static void *mem_alloc(size_t size, uint8_t type)
 {
 	void *ptr = NULL;
+	size_t cacheline_size = dcache_get_line_size();
```

Comment:
The return value of `dcache_get_line_size()` is only `if (type & MEM_TYPE_ALIGN)`, please move the function call to that block.

Reply:
Fixed in https://github.com/OP-TEE/optee_os/pull/4407

Reply:
Fixed in https://github.com/OP-TEE/optee_os/pull/4407